### ISSUE TITLE
Breaks on Chromium 39.0.2166.0

### ISSUE THREAD

**Issue by DomT4** [uBlock-LLC/uBlock#244]
Chromium Version 39.0.2166.0 for OS X and uBlock 0.6.30 do not seem to be a winning combination at this point. 

The last version of uBlock worked fine, but I've been having big trouble since 0.6.30 for some reason. Whenever I try to open up the uBlock extension page it hangs indefinitely and is greeted by the Chromium light blue/dark blue spinning wheel of death.

You can click through the rest of the uBlock extension page tabs fine, but the third-party filters page is completely greyed out and you can't touch any of the boxes, and nor do any of the third-party filters available actually load.

Is there any special debug reports you need on this or such?

**Comment by gorhill** [comment]
Looks like network issues. The dev console should have more details.

**Comment by gorhill** [comment]
Problem is that currently I am not setup to test non-stable version of Chromium, not sure when I will have the time to setup all this-- I need qemu, etc.

**Comment by gorhill** [comment]
I am looking at the code, and it should not stay in that spinning wheel mode forever, there is a timeout of 15s for when there are network issues. I think there are two files downloaded from remote servers if you have _Auto update_ enabled, which means this could amount to 30s "freeze" if there are network issues.

**Comment by ghost** [comment]
On Archlinux (or Manjaro) you could have both stable `chromium` and dev chromium (as `chromium-dev` command) each with their own folders. The only thing is that you'd have to recompile the dev version from zero every time you update it (releases like once a week; but all this is done automatically EDIT: after you initiate it  - it just takes time and space while it compiles).
Also the rolling release style(of these distros) may be worth it, vs. linux mint that is.

**Comment by DomT4** [comment]
For some reason, this magically stopped happening when I fired Chromium back up a minute ago. All seems to be working fine again now. Strange.

The delay was indefinite last night. I went away and left the page open whilst I did a little tidying, etc and came back 20 minutes later and the Chromium loading ball of death was still spinning.

I'll close this out for now, but if I keep hitting recurring issues at specific times of day I'll open it again and try and collect some more information for you.
